src/stan/lang/ast/node/cholesky_corr_var_decl.hpp

```diff
@@ -33,10 +33,12 @@ namespace stan {
        * @param K number of rows and columns
        * @param name variable name
        * @param dims array dimension sizes
+       * @param def defition of variable
        */
       cholesky_corr_var_decl(const expression& K,
                              const std::string& name,
-                             const std::vector<expression>& dims);
+                             const std::vector<expression>& dims,
+                             expression const& def);
```

Comment:
`const expression& def`

Means the same, but it'll be consistent.  This is another one of those space-vs-tabs things, but we just decided to go this way for reasons I don't even recall.  I tried to update all the AST code as I went through.